Makefile

```
@@ -175,20 +181,20 @@ bench_tsdb: $(PROMU)
 	@$(GO) tool pprof -svg $(PROMTOOL) $(TSDB_BENCHMARK_OUTPUT_DIR)/mutex.prof > $(TSDB_BENCHMARK_OUTPUT_DIR)/mutexprof.svg
 
 .PHONY: cli-documentation
-cli-documentation:
+cli-documentation: ## Generate Prometheus and promtool command-line documentation
 	$(GO) run ./cmd/prometheus/ --write-documentation > docs/command-line/prometheus.md
 	$(GO) run ./cmd/promtool/ write-documentation > docs/command-line/promtool.md
 
 .PHONY: check-go-mod-version
-check-go-mod-version:
+check-go-mod-version: ## Verify Prometheus go.mod version consistency
 	@echo ">> checking go.mod version matching"
 	@./scripts/check-go-mod-version.sh
 
 .PHONY: update-all-go-deps
-update-all-go-deps:
+update-all-go-deps: ## Update all Prometheus Go dependencies including examples
```

personally I would use full sentence (with trailing period), but not a blocker.
```suggestion
update-all-go-deps: ## Update all Prometheus Go dependencies including examples.
```